src/main/pig/EnhanceColumn.pig

```diff
@@ -0,0 +1,44 @@
+/**
```

Comment:
Sorry, can we make such pig merged together with stats.pig, two reasons:
1. Save jobs and improve performance
2. InitStep and InitModelProcessor is not good for such enhance jobs since after init before stats usally users config column related configurations.